diff --git a/node_modules/primeng/fesm2022/primeng-inputnumber.mjs b/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
index 04756cf..4c896d2 100644
--- a/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
+++ b/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
@@ -14,6 +14,7 @@ import { TimesIcon, AngleUpIcon, AngleDownIcon } from 'primeng/icons';
 import { InputText } from 'primeng/inputtext';
 import { style as style$1 } from '@primeuix/styles/inputnumber';
 import { BaseStyle } from 'primeng/base';
+import { DomHandler } from 'primeng/dom';
 export * from 'primeng/types/inputnumber';
 
 const style = /*css*/ `
@@ -573,7 +574,7 @@ class InputNumber extends BaseInput {
         if (max && max < this.formatValue(newValue).length) {
             return;
         }
-        this.updateInput(newValue, null, 'spin', null);
+        !DomHandler.isTouchDevice() && this.updateInput(newValue, null, 'spin', null);
         this.updateModel(event, newValue);
         this.handleOnInput(event, currentValue, newValue);
     }
@@ -588,7 +589,6 @@ class InputNumber extends BaseInput {
             return;
         }
         if (!this.$disabled()) {
-            this.input?.nativeElement.focus();
             this.repeat(event, null, 1);
             event.preventDefault();
         }
@@ -619,7 +619,6 @@ class InputNumber extends BaseInput {
             return;
         }
         if (!this.$disabled()) {
-            this.input?.nativeElement.focus();
             this.repeat(event, null, -1);
             event.preventDefault();
         }
