diff --git a/node_modules/primeng/fesm2022/primeng-inputnumber.mjs b/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
index e6ca869..6ac9c2f 100644
--- a/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
+++ b/node_modules/primeng/fesm2022/primeng-inputnumber.mjs
@@ -10,6 +10,7 @@ import { BaseComponent } from 'primeng/basecomponent';
 import { TimesIcon, AngleUpIcon, AngleDownIcon } from 'primeng/icons';
 import { InputText } from 'primeng/inputtext';
 import { BaseStyle } from 'primeng/base';
+import { DomHandler } from 'primeng/dom';
 
 const theme = ({ dt }) => `
 .p-inputnumber {
@@ -775,7 +776,7 @@ class InputNumber extends BaseComponent {
         if (this.maxlength && this.maxlength < this.formatValue(newValue).length) {
             return;
         }
-        this.updateInput(newValue, null, 'spin', null);
+        !DomHandler.isTouchDevice() && this.updateInput(newValue, null, 'spin', null);
         this.updateModel(event, newValue);
         this.handleOnInput(event, currentValue, newValue);
     }
@@ -790,7 +791,7 @@ class InputNumber extends BaseComponent {
             return;
         }
         if (!this.disabled) {
-            this.input?.nativeElement.focus();
+            this.autofocus && this.input?.nativeElement.focus();
             this.repeat(event, null, 1);
             event.preventDefault();
         }
@@ -821,7 +822,7 @@ class InputNumber extends BaseComponent {
             return;
         }
         if (!this.disabled) {
-            this.input?.nativeElement.focus();
+            this.autofocus && this.input?.nativeElement.focus();
             this.repeat(event, null, -1);
             event.preventDefault();
         }
